app/operations/concerns/url_labels.rb

```diff
@@ -1,5 +1,5 @@
 module UrlLabels
-  def extract_url_labels(urls)
+  def self.extract_url_labels(urls)
```

Comment:
Remove the include (projects controller, others?) as it's directly called now in `ContentFromParams`